tests/Database/DatabaseEloquentModelTest.php

```diff
@@ -1924,7 +1943,7 @@ public function testDecrementQuietlyOnExistingModelCallsQueryAndSetsAttributeAnd
         $model->publicDecrementQuietly('foo', 1, ['category' => 1]);
         $this->assertEquals(2, $model->foo);
         $this->assertEquals(1, $model->category);
-        $this->assertTrue($model->isDirty('category'));
+        $this->assertFalse($model->isDirty('category'));
```

Comment:
I changed this behaviour since it's not expected that the property is still dirty after update